Assets/MRTK/SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -1,11 +1,11 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-using Microsoft.MixedReality.Toolkit.Input;
-using Microsoft.MixedReality.Toolkit.Utilities;
 using System;
 using System.Collections.Generic;
 using System.Runtime.CompilerServices;
+using Microsoft.MixedReality.Toolkit.Input;
```

Comment:
Nit: Move this back up to keep using statements alphabetical

Reply:
Quick VS tip

You can sort the using in VS with this:
![image](https://user-images.githubusercontent.com/53493796/84825117-cbc80680-afd5-11ea-84e8-eb419aec0207.png)

Reply:
Heads up, `dotnet-format` doesn't seem to agree.

`dotnet-format` aggressively recommends putting System* namespaces first, then alphabetically sorting the rest of the namespaces.